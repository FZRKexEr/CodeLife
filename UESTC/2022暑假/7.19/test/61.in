10
qbinrtvrcxvnnqsktxkpeydksglknoxadsaujqqpuiehtactuubeefucwgaumzdtcvxijpbumbdvtwtpwsefwdhmjvitquxylzns
1
hrowhpsqjiehffnchvvdzjenazitaeisuhcavipanfyozmwdrpqjycrmfipvnqgaxcluvhbfuodacgukzojkdgchqqnjyuxsypqm
1
hjocvtvdznetvukromlkthbiaywzzbwqksffezkuytavddkrjqwzknluzweibwbtydgzdpysucqvxlnzkwxpgzumoxjnfimlwpkc
10
hxotopxrvnoqutdmwvvbsatltgdzkoiwcicfvskmosnbfxqzmgnvttwyecqejoqpisemmmuxcjhgszaenapyytabkedzqbivfsla
4
jbqgmvnqybmbhhueymfzkuvvhegitdcojvimyoaocbzwwftnojdqkvazlpqeknsfcuhxhcmrqzexhnnbwsmotbcqrbavhlfwndrj
7
oedhlaiaarijjfjyhmuplucvysamsjkozfzxvrgjrulvdkbggmnnoreewpqcdmctqtrvgsrlpvjcrjktvvsfasjyigcnmdxycuir
10
gbbrtvbpxcdasrcaggjxgypzntroszvwqxbkwlwefqhbbedldwboqajuzkuelpxjkpwcrnrcrnatvjiiggtypxauzxdebwbpcelc
2
mmmmfhwcmqeqfwjtcoeyckcvhzfrsilidgsdbgdyagyxsgpxgeucgrrczwxqqllcldjychkfyplcuczbxuyegfdkbdjakdyubpjx
5
bfmtpeomyixrlthdpisxpyfhiwlrqowlbqftnnkyxhefprgbympptwqarpksubizsrikynwntbqzifraxggnjelnpcovpxeksbhs
8
ohllklpwciovmbfxsvuhzvpckilhwwbykglzfvaoiwpflbrjwuzccncuffttpmsswsdntkdjmgfrruwcqlhjzmwcjjynvrxnxqdc
8
