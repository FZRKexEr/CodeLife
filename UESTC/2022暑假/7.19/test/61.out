882499718
882499718
1024
33554432
16384
1024
65980984
1048576
4096
4096
